tools/xcodeproj_shims/installers/lldb-settings.sh

```diff
@@ -15,8 +15,26 @@ set -euo pipefail
 #
 #     command source ~/.lldbinit-source-map
 cat <<-END > ~/.lldbinit-source-map
-settings set target.source-map ./external/ "$BAZEL_WORKSPACE_ROOT/bazel-$(basename "$BAZEL_WORKSPACE_ROOT")/external"
-settings append target.source-map ./ "$BAZEL_WORKSPACE_ROOT/"
+settings set target.source-map ./ "$BAZEL_WORKSPACE_ROOT/"
 settings set target.sdk-path $SDKROOT
 settings set target.swift-framework-search-paths $FRAMEWORK_SEARCH_PATHS
 END
+
+set +u
+if [[ -n $BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS ]]
```

Comment:
could do `-n ${BAZEL_LLDB_SWIFT_EXTRA_CLANG_FLAGS:-}` so you don't need to turn off `-u`

Reply:
thank you!! i was having trouble figuring this out.